src/lib/__tests__/date.test.ts

```diff
@@ -15,7 +15,7 @@ import {
 } from "lib/date"
 
 describe("date formatting", () => {
-  describe(timeRange, () => {
+  describe("timeRange", () => {
```

Comment:
def think this form is a bit weird and unexpected and clever, but its also valid since it will read the `function.name` prop. Did this throw a warning / error?

Reply:
`Title must be a string (eslintjest/valid-title)`

Reply:
Non blocking, especially after all of this work was done, but i think that rule could be disabled. I can't remember where i read it but there's some interesting introspection benefits around passing the function in versus a string, but i also wonder about eslintjest setting this rule as a default -- in like a what do they know kind of way /shrug

Reply:
I'd be curious to know what the benefits are, if you ever dig it up. I'm inclined to roll with the string form since it's just more consistent and the default behavior of the recommended settings. There's a bit of discussion I dug up here https://github.com/jest-community/eslint-plugin-jest/issues/431 for context as well.